csrankings-i.csv

```diff
@@ -368,10 +369,10 @@ Itaru Kitahara,University of Tsukuba,https://sites.google.com/image.iit.tsukuba.
 Itsik Pe'er,Columbia University,http://www.cs.columbia.edu/~itsik,l3KIq-MAAAAJ
 Ittay Eyal,Technion,http://webee.technion.ac.il/people/ittay,1oUGY7cAAAAJ
 Iulian Neamtiu,NJIT,https://web.njit.edu/~ineamtiu,8qU-5YMAAAAJ
-Ivan Aaen,Aalborg University,http://people.cs.aau.dk/~ivan,-8-9oRgAAAAJ
+Ivan Aaen,Aalborg University,http://people.cs.aau.dk/~ivan,89oRgAAAAJ

```

Comment:
Your editor apparently deleted the leading hyphen from this and the below Google Scholar IDs. I'd be interested to know how this happened so we can prevent it from happening in the future!